src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,34 @@
 import React from 'react';
-import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Film {
+  [key: string]: string;
+}
```

Comment:
```suggestion
interface Film {
  [key: string]: string;
}
```
Let's describe all fields of this object (title. description, imdbId etc.) and move this interface to a separate file (like `src/types/Film.tsx`)